qtconsole/console_widget.py

```diff
@@ -1833,6 +1833,8 @@ def _insert_plain_text(self, cursor, text, flush=False):
                         self._set_top_cursor(cursor)
                         cursor.joinPreviousEditBlock()
                         cursor.deletePreviousChar()
+                        cursor.select(QtGui.QTextCursor.Document)
```

Comment:
Let's enclose this in a `if os.name == 'nt'` statement because it could cause unknown issues in other platforms.